fossunited/foss_hackathon/doctype/foss_hackathon_localhost/test_foss_hackathon_localhost.py

```diff
@@ -0,0 +1,169 @@
+import frappe
+import frappe.permissions
+from frappe.tests import IntegrationTestCase
+
+from fossunited.doctype_ids import (
+    HACKATHON_LOCALHOST,
+    USER_PROFILE,
+)
+from fossunited.tests.utils import insert_test_hackathon, insert_test_hackathon_localhost
+
+
+class TestFOSSHackathonLocalHost(IntegrationTestCase):
+    def setUp(self):
+        self.ORGANIZER_1 = "test1@example.com"
+        self.organizer1_profile = frappe.get_doc(USER_PROFILE, {"user": self.ORGANIZER_1})
+        self.ORGANIZER_2 = "test2@example.com"
+        self.organizer2_profile = frappe.get_doc(USER_PROFILE, {"user": self.ORGANIZER_2})
```

Comment:
maybe stupid question - don't we need to save the profiles explicitly? i thought this returns a new document that isn't stored to disk yet.

Reply:
so, test users which i usually write (test1@...., test2@...) are out of box from frappe. 
With our logic for user creation, profiles are auto created when users are created.
We do not need to create profiles for these test users as those are already created when these users are created. 

We just need to `get` them to use in tests. Unless doing explicit update operations